src/script/lua_api/l_server.cpp

```diff
@@ -505,6 +515,8 @@ void ModApiServer::Initialize(lua_State *L, int top)
 	API_FCT(get_modpath);
 	API_FCT(get_modnames);
 
+	API_FCT(print);
```

Comment:
Its removed again in `builtin/init.lua`. I can remove it if you want, and document it in lua_api.txt.